distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -619,6 +619,10 @@ logging:pattern=%date{ISO8601} - %-25logger: %msg%n
 # ScheduledExecutorService. (optional, defaults to 5)
 #knx:scheduledExecutorServiceShutdownTimeoutString=
 
+# Use NAT (Network Address Translation)
+#  (optional; defaults to false)
+#knx:useNAT=
```

Comment:
At the risk of seeming too picky, if this showed the commented-out `true` setting, then it would not be left to assume that `true` was the precise thing to enter here when the user uncomments the line.